extensions/big-data-cluster/tsconfig.json

```diff
@@ -1,15 +1,23 @@
 {
-	"extends": "../shared.tsconfig.json",
+	"compileOnSave": true,
 	"compilerOptions": {
```

Comment:
Why are you changing these options?

Reply:
not changing. basically replaced with new file.

Reply:
Yes, but why did you do that? Generally using the shared config is preferable.